tests/protocol/dns/basic.zeek

```diff
@@ -3,7 +3,15 @@
 # @TEST-EXEC: ${ZEEK} -r ${TRACES}/dns53.pcap %INPUT
 # @TEST-EXEC: btest-diff conn.log
 # @TEST-EXEC: btest-diff dns.log
+# @TEST-EXEC: if zeek-version 32000; then btest-diff .stdout; fi
```

Comment:
Hmm, did you intend to perform this check _exclusively for 3.2_?  All the other checks here are for >=3.2 (assuming below `Version::number >= 32000` is equivalent to `ZEEK_VERSION >= 30200` in the evt file).

Maybe we could alternatively hardcode the expected output for versions < 3.2.

Reply:
The script checks for `>=`  on its end if it's used like that. But turns out I forgot to commit the script. It's the same as in `spicy-plugin`.

Reply:
Ah, I see 🤦